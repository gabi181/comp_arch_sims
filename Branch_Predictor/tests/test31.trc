8 4 15 3 local_history global_tables using_share_lsb
0x2261C848 T 0x16496EB0
0x1140F1F0 N 0x4826F768
0x1140F1F0 N 0x4826F768
0x1140F1F0 T 0x4826F768
0x744650D8 T 0x5B979D74
0x720828A8 T 0x798FB308
0x2261C848 T 0x16496EB0
0x744650D8 T 0x5B979D74
0x63F8AA30 T 0x2FC1E19C
0x4C0DA938 T 0x643032E0
0x744650D8 N 0x5B979D74
0x4C0DA938 T 0x643032E0
0x4C0DA938 T 0x643032E0
0x4C0DA938 N 0x643032E0
0x2261C848 T 0x16496EB0
0x4C0DA938 N 0x643032E0
0x720828A8 T 0x798FB308
0x63F8AA30 N 0x2FC1E19C
0x4C0DA938 N 0x643032E0
0x63F8AA30 T 0x2FC1E19C
0x4C0DA938 T 0x643032E0
0x2261C848 T 0x16496EB0
0x720828A8 N 0x798FB308
