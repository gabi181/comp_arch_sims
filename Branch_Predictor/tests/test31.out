0x2261c848 N 0x2261c84c
0x1140f1f0 N 0x1140f1f4
0x1140f1f0 T 0x4826f768
0x1140f1f0 N 0x1140f1f4
0x744650d8 N 0x744650dc
0x720828a8 N 0x720828ac
0x2261c848 N 0x2261c84c
0x744650d8 T 0x5b979d74
0x63f8aa30 N 0x63f8aa34
0x4c0da938 N 0x4c0da93c
0x744650d8 N 0x744650dc
0x4c0da938 N 0x4c0da93c
0x4c0da938 T 0x643032e0
0x4c0da938 T 0x643032e0
0x2261c848 T 0x16496eb0
0x4c0da938 T 0x643032e0
0x720828a8 N 0x720828ac
0x63f8aa30 T 0x2fc1e19c
0x4c0da938 T 0x643032e0
0x63f8aa30 T 0x2fc1e19c
0x4c0da938 T 0x643032e0
0x2261c848 N 0x2261c84c
0x720828a8 N 0x720828ac
flush_num: 15, br_num: 23, size: 432b
